Source/WebKit/UIProcess/skia/BackingStoreSkia.cpp

```diff
@@ -29,34 +29,93 @@
 #if USE(SKIA)
 
 #include "UpdateInfo.h"
-#include <WebCore/NotImplemented.h>
+#include <WebCore/GraphicsContextSkia.h>
+#include <skia/core/SkBitmap.h>
+#include <skia/core/SkCanvas.h>
 
 namespace WebKit {
 
-static const Seconds s_scrollHysteresisDuration { 300_ms };
+static sk_sp<SkSurface> createSkiaImageSurfaceWithFastMalloc(const WebCore::IntSize& size, double deviceScaleFactor)
```

Comment:
Note that we are already using a custom allocator for skia that uses bmalloc, so you don't need to explicitly alloc data.